app/views/account/profiles/_form.html.slim

```diff
@@ -0,0 +1,8 @@
+= simple_form_for @user, url: [:account, :profile] do |f|
+  = f.input :first_name, wrapper: :bulma_input, required: true
+  = f.input :last_name, wrapper: :bulma_input, required: true
+  = f.input :role, wrapper: :bulma_input
+  = f.input :department, wrapper: :bulma_input
+  = f.input :about, wrapper: :bulma_input
```

Comment:
dont wrap textarea as input 
check bulma documentation, they have own class for textarea and new wrapper is needed